src/Fmt_ast.ml

```diff
@@ -1252,10 +1261,15 @@ and fmt_expression c ?(box= true) ?eol ?parens ?ext ({ast= exp} as xexp) =
                  $ hvbox (-2) (list flds "@,; " fmt_field) ) ))
         $ fmt_atrs )
   | Pexp_sequence _ ->
+      let pp_sep =
+        match ext with
+        | None -> fun fs () -> Format.fprintf fs " ;@ "
+        | Some {txt; _} -> fun fs () -> Format.fprintf fs " ;%%%s@ " txt
```

Comment:
A minor point, but it would be more consistent with the existing code to change this to e.g.:
```
| None -> fmt " ;@ "
| Some {txt; _} -> fmt " ;%%" $ str txt $ fmt "@ "
```
and probably name it `fmt_sep` instead of `pp_sep`.